conans/client/generators/scons.py

```diff
@@ -12,7 +12,9 @@ def content(self):
                     '        "CPPPATH"     : {info.include_paths},\n'
                     '        "LIBPATH"     : {info.lib_paths},\n'
                     '        "BINPATH"     : {info.bin_paths},\n'
-                    '        "LIBS"        : {info.libs},\n'
+                    '        "LIBS"        : {libs},\n'
```

Comment:
If we are simplifying the logic in the generator classes, them it's better to write here and do not modify the python code.

```
"LIBS"     : {info.libs},{info.system_libs},\n
```

Reply:
This was introducing a double comma if system_libs is empty, I wasn't sure if SCons is going to process that correctly

Reply:
To be honest, using the _repr_ representation of  `info.*` is very risky, it will change if using a tuple and will break for sure if using a generator.

What about changing it to a jinja2 template and be explicit about the rendering?

```
[{{ ', '.join(info.libs) + list(info.system_libs)) }}]
```

(or using the `|join(", ")`)